Support custom `ffmpeg` path for `YouTube Downloader` extension

vimtor opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

Closes #7230 #7383

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

thomaslombart reviewed (commented):
Could you add an entry in the `CHANGELOG.md` file? Here's our guidelines about that: [Version History](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history).

vimtor commented:
Hey @thomaslombart, I've added a new changelog entry and modified the code to support custom `ffmpeg`. This closes #7383 and resolves all related issues.
